socorro/schemas/crash_report.json

```diff
@@ -307,6 +307,10 @@
                 "crashing_thread": {
                     "type": "object",
                     "properties": {
+                        "thread_name": {
+                            "type": ["string", "null"],
+                            "description": "Name of the crashing thread."
+                        },
```

Comment:
The changes to this file aren't needed if you're just adding a field for querying to Super Search.